torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -195,6 +195,59 @@ c10::optional<at::Tensor> runTorchSlice_opset10(
   return c10::optional<at::Tensor>(updated_val);
 }
 
+// Refer to AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_EXCEPT_COMPLEX_HALF
+at::Tensor runTorchArange_opset11(
+    const Node* node,
+    const std::vector<at::Tensor>& inputTensorValues) {
+  AT_ASSERT(inputTensorValues.size() == 3);
+  auto dtype = inputTensorValues[0].scalar_type();
+  at::Tensor updated_val;
+  switch (dtype) {
+    case at::ScalarType::Float: {
+      auto start = inputTensorValues[0].item<float>();
+      auto end = inputTensorValues[1].item<float>();
+      auto step = inputTensorValues[2].item<float>();
+      updated_val = at::arange(start, end, step);
+      break;
+    }
+    case at::ScalarType::Double: {
+      auto start = inputTensorValues[0].item<double>();
+      auto end = inputTensorValues[1].item<double>();
+      auto step = inputTensorValues[2].item<double>();
+      updated_val = at::arange(start, end, step);
```

Comment:
can this line be moved out of switch statement and added at the end of the function, before return updated_val

Reply:
Seems not, `start`, `end`, `step` are all type dependent, and defined in the inner {}, so we update in this inner {} also.